bundles/org.openhab.binding.rfxcom/src/main/java/org/openhab/binding/rfxcom/internal/messages/RFXComLighting1Message.java

```diff
@@ -229,6 +230,10 @@ public void convertFromState(String channelId, Type type) throws RFXComUnsupport
                 }
                 break;
 
+            case CHANNEL_COMMAND_STRING:
+                command = RFXComLighting1Message.Commands.valueOf(type.toString().toUpperCase());
```

Comment:
Is the prefix "RFXComLighting1Message" necessary? Just wondering because you did not use in Lighting5.

Reply:
Nope, was caused by copy-pasting and Intelij acting smart 😄